docs/faq/product.md

```diff
@@ -6,7 +6,7 @@ description: Questions related to product and design.
 
 ### Can we call Backstage something different? So that it fits our company better?
 
-Yes, Backstage is just a platform for building your own developer portal. We
+Yes, Backstage is just a open source framework for building your own developer portal. We
```

Comment:
I suggest removing "open source" here, so that it reads: 
`Yes, Backstage is just a framework for building your own developer portal.`

(Otherwise, if keeping it as "open source framework", then it should be "an", not "a" — `an open source framework`.)